src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -316,6 +316,24 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    /*
+     * returns true if the session is closed within the past hour of calling this function
+     */
+    
```

Comment:
This will not be a javadoc comment: it should start with `/**` instead of `/*`.
Also, the empty space should be before the header comment, not after.